site/profile/manifests/reverse_proxy.pp

```diff
@@ -3,6 +3,7 @@
   Hash[String, String] $subdomains,
   Hash[String, Array[String]] $remote_ips = {},
   String $main2sub_redir = 'jupyter',
+  String $robots_txt = file('profile/reverse_proxy/robots.txt'),
```

Comment:
If we don't create a local robots.txt file, we can specify the content directly:
```String $robots_txt = 'User-agent: *\nDisallow: /',```